packages/Calendar/tests/DateRangeCalendar.cypress.js

```diff
@@ -47,10 +47,10 @@ describe("<DateRangeCalendar />", () => {
     cy.findByTestId("calendar.shortcut").should("be.visible");
 
     cy.get('label[for$="-0"]').click();
-    cy.get('label[for$="-2018"]').click();
+    cy.get('label[for$="-2024"]').click();
```

Comment:
All of these are not needed for my change, but looks like making the PR in 2024 was the issue. When running locally I can confirm that the 2018 year is now pushed out of view by the current 2024 date, so I updated it to this for now (should give another 5 years or so 😅)

Local run: 
![SingleDateCalendar -- should open shortcut panel and be able to jump to target month (failed)](https://github.com/acl-services/paprika/assets/1757057/5349529e-2f86-4d3d-850e-489b059431c2)